scalableminds/webknossos
# Scroll faster through tree comments

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open the tracing view
- create/have many trees with comments
- go to the comment tab and scroll though them with n and p
- holding the keys pressed to scroll permanent
- the delay between pressing the key and the actual start of the scrolling should match the users' delay

### Issues:
- fixes [#2924](https://github.com/scalableminds/webknossos/issues/2924)

------
